pages.fr/common/ifconfig.md

```diff
@@ -0,0 +1,24 @@
+# ifconfig
+
+> Configurateur des interfaces réseau.
+
+- Affiche les paramètres de réseau d'un adaptateur ethernet :
+
+`ifconfig eth0`
```

Comment:
This is a bug in the English page, the `eth0` should be between `{{}}`:
```suggestion
`ifconfig {{eth0}}`
```
You can add the fix here if you want.